src/pages/trader-tools/pip-calculator/_pip-calculator.js

```diff
@@ -0,0 +1,437 @@
+                            <Localize
+                                translate_text="<0>Pip value = point value × volume × contract size</0>"
+                                components={[<strong key={0} />]}
+                            />
+                        </Text>
+
+                        <Text mb="1.6rem">
+                            <Localize translate_text="For synthetic accounts, the pip value is calculated in USD." />
+                        </Text>
+                        <Text mb="40px">
+                            {localize(
+                                'For financial accounts, the pip value is in the quote currency for forex pairs.',
+                            )}
+                        </Text>
+
+                        <Header as="h3" type="section-title" mb="0.8rem">
+                            {localize('Example calculation')}
+                        </Header>
+
+                        <Accordion has_single_state>
+                            <AccordionItem
+                                header={localize('Pip value for synthetic indices')}
+                                header_style={header_style}
+                                style={item_style}
+                                plus
+                            >
+                                <Text mb="16px">
+                                    {localize(
+                                        'Let’s say you want to trade 1 lot of Volatility 75 Index.',
+                                    )}
+                                </Text>
+
+                                <Show.Desktop>
+                                    <QueryImage
+                                        data={data.pip_value_formula}
+                                        alt={localize('Margin formula')}
+                                    />
+                                    <QueryImage
+                                        data={data.pip_info}
+                                        alt={localize('Pip Info')}
+                                    />
+                                </Show.Desktop>
+
+                                <Show.Mobile>
+                                    <QueryImage
+                                        data={data.pip_value_formula_mobile}
+                                        alt={localize('Margin formula mobile')}
+                                    />
+                                    <QueryImage
+                                        data={data.pip_info_mobile}
+                                        alt={localize('Pip Info')}
+                                    />
+                                </Show.Mobile>
+
+                                <Text mb="1.6rem" mt="1.6rem">
+                                    <Localize
+                                        translate_text="So your pip value is 0.01 USD."
+                                        components={[<strong key={0} />]}
```

Comment:
I don't see the `strong` tag being used 👀  We can replace this with a regular `localize` instead.